### ISSUE TITLE
Issue with Token 

### ISSUE THREAD

**Issue by KonstantinGOT** [facebookincubator/Facebook-For-OpenCart#253]
Hi have a issue with token when i go in my admin. when i go to extension is says soemthing bad (look screen)  also i dont see the token in advance settings 
![Screen Shot 2019-09-20 at 11 16 34](https://user-images.githubusercontent.com/50840258/65349406-28f27100-db98-11e9-89fa-83d01e67627a.png)
![Screen Shot 2019-09-20 at 11 12 25](https://user-images.githubusercontent.com/50840258/65349411-2abc3480-db98-11e9-9899-e8c276f26c20.png)

**Comment by shue249** [comment]
@KonstantinGOT Can you look at this post and see if the steps help you - https://github.com/facebookincubator/Facebook-For-OpenCart/issues/142#issuecomment-482908951

**Comment by KonstantinGOT** [comment]
For short time yea but then again same issue..

**Comment by shue249** [comment]
@KonstantinGOT We are now working on a revamp of the catalog sync to ignore the use of the API and access token. We will update you once this is ready.

**Comment by shue249** [comment]
@KonstantinGOT  

We have released a new version of the plugin, 2.2.0. In this new version, we have changed the catalog sync to using 100% feed file and the API/access token is no longer needed. This will resolve your current problem. You need to do an upgrade of the existing catalog and the steps to do that is highlighted in the release notes.

Here is the release notes:
2.2.0 - 11 Nov 2019
* Changed the product catalog sync method. On earlier versions, the plugin adopted 2 approaches - using feed file for initial catalog sync and API for subsequent product changes. From this version onwards, the plugin is adopting feed file for both initial catalog sync as well as subsequent product changes.
  1. There will be a daily feed pull to update the product catalog changes from your OpenCart website to Facebook product catalog.
  2. For existing plugin merchants, you need to perform a step to automatically generate the daily pull mechanism. Access the Facebook Business Extension module, click on Manage Settings, click on Advanced options. You will be able to see the notification indicating that the Catalog sync is successful. See screenshot here - https://drive.google.com/open?id=1dAF88Spg-iAwPXmkwcVvAqg82UYJ5IJ0
  3. Access token is no longer needed and there is no longer any need to update the access token nor to keep the Facebook account signed in.
  4. Better integration with other 3rd party product plugins. You can use any of the 3rd party product plugins to perform add/edit/delete product changes. These product changes will be automatically updated to Facebook via the daily feed mechanism.
  5. You can trigger immediate product sync to Facebook. Access the Facebook Business Extension module, click on Manage Settings, click on Your Products. Click on Fetch Now to sync your products to Facebook. See screenshot here - https://drive.google.com/open?id=1bCTT4gU4gC8pi6T6HLhpe2fkHWI-RvhN

We will be closing out this ticket, feel free to reach to us on this ticket if you have any further questions - https://github.com/facebookincubator/Facebook-For-OpenCart/issues/302
